### PR TITLE
174710980 navigate back on questionnaire

### PR DESCRIPTION
**Pull request by johols**
Implemented a back button in Questionnarie. 

About the logic: the idea is to use the same logic as changing one question forward, but in reverse.

A separate component holds the "view" part which uses props **text**  and **clickPrevHandler**

### CODE REVIEW

File: src/__tests__/src/__components__/PrevButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import PrevButton from '../../../components/PrevButton';
+
+const title = 'Previous';
+const clickPrevHandler = jest.fn();
+
+describe('ExpandableCard', () => {
```
Comment (NickCallaghan) [inline src/__tests__/src/__components__/PrevButton.test.tsx:8]: Describes the prevButton not expandable card

File: src/assets/icon-arrow-back.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (NickCallaghan) [inline src/assets/icon-arrow-back.svg:1]: No biggie but it think this is already included in MUI: 
import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
https://material-ui.com/components/material-icons/

File: src/components/PrevButton.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        </div>
+      </div>
+    );
+}
+
+PrevButton.defaultProps = {
+    text: 'Back'
+}
+
+export default PrevButton;
```
Comment (NickCallaghan) [inline src/components/PrevButton.tsx:54]: We should probably have a way to make the button disabled. For example on the first question

**Review by NickCallaghan — APPROVED** [review]

File: src/components/PrevButton.tsx (hunk 3 above)
Comment (fe-mason) [reply src/components/PrevButton.tsx:54]: Could we hide the button on Q1 instead of disabling? Or make it route back to /personality?

**Pull request merged**